src/crux/codec.clj

```diff
@@ -312,11 +312,11 @@
 
 (defmethod print-method Id [id ^Writer w]
   (.write w "#crux/id ")
-  (.write w (str id)))
+  (.write w (pr-str (str id))))
```

Comment:
this doesn't fix anything we know of yet, doesn't break any existing tests either, just more consistency
but I'm open to hear more test ideas on this one